core/src/main/java/org/infinispan/marshall/persistence/PersistenceMarshaller.java

```diff
@@ -23,6 +24,8 @@
     */
    void register(SerializationContextInitializer initializer);
 
+   ImmutableSerializationContext getSerializationContext();
```

Comment:
It is only unused because I had to do the hacky thing to get the user context. If this is the way I should be doing it, then I am fine removing it. https://github.com/infinispan/infinispan/pull/9480/files#diff-05c73e851df1a197533a5e98b9d28ed7a95b31ead0ff6dadc09e4d23e563fd23R77